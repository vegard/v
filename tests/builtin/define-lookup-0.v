# prints 0, should be error
x := u64 10;
@print x;
