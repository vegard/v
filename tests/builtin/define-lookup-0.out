tests/builtin/define-lookup-0.v:3:7: cannot access run-time variable x at compile time
@print x;
       ^
